DB-week1.sql

```diff
@@ -0,0 +1,29 @@
+--Find out how many tasks are in the task table
+ select count(*) from task;
+--Find out how many tasks in the task table do not have a valid due date
+select count(*) from task where due_date is null;
+--Find all the tasks that are marked as done
+select t.* from task t    
+   left join status s on s.id=t.status_id
+   where s.name='Done';
+--Find all the tasks that are not marked as done
+select t.* from task t
+   left join status s on s.id=t.status_id
```

Comment:
you can have tasks without status that will appear on the result, please check this query again

Reply:
Hi @jpganz, although I did not find any task without status Id but I tried to solve the task , please check it.